kong/db/strategies/off/tags.lua

```diff
@@ -1,11 +0,0 @@
-local Tags = {}
-
--- Used by /tags/:tag endpoint
```

Comment:
Meaning that in declarative mode you can't do this anymore, right? Is this a breaking change?

Reply:
As we talked on meeting, tags is a undocumented feature, we think that removing it will be safe.